tests/source/comment.rs

```diff
@@ -33,6 +33,42 @@ fn test() {
 
     funk(); //dontchangeme
             // or me
+
+    // #1388
+    const EXCEPTION_PATHS: &'static [&'static str] =
```

Comment:
I think this test could be shorter - I assume the bug reproduces with a smaller test case?